messages/install.txt

```diff
@@ -33,12 +33,12 @@ If you are on windows, or don't usually use git from the command
 line, there might be some things you'll need to set up. Go here
 for instructions on how to get set up:
 
-    https://docs.sublimegit.net/quickstart.html
+    https://github.com/SublimeGit/SublimeGit/blob/master/docs/quickstart.rst
 
 
 Issues, Questions or Bugs?
 --------------------------
 Send a mail to support@sublimegit.net, or report it in the issue
 tracker:
 
-    https://github.com/SublimeGit/SublimeGit/issues
+    https://github.com/SublimeGit/SublimeGit/issues/new
```

Comment:
I don't think sending people directly to the new issue is a good idea. People should try and look over existing issues before filing a new one.